VisaCheckout/src/test/java/com/braintreepayments/api/test/TestConfigurationBuilder.java

```diff
@@ -1,7 +1,5 @@
 package com.braintreepayments.api.test;
 
-import com.braintreepayments.api.models.GraphQLConfiguration;
```

Comment:
Changes in this file could be a separate PR. This configuration builder had too much detail to it. We only need to build a Visa Checkout configuration.